courses/free-courses-en.md

```diff
@@ -530,6 +532,7 @@
 * [Intro to Machine Learning](https://www.kaggle.com/learn/intro-to-machine-learning) (Kaggle)
 * [Intro to Machine Learning Udacity](https://www.udacity.com/course/intro-to-machine-learning--ud120) (Udacity)
 * [Machine Learning](https://www.sololearn.com/Course/machine-learning) (SoloLearn) *(account required)*
+* [Machine Learning Mastery With Python](https://docplayer.net/60629865-Machine-learning-mastery-with-python.html) - (Book) 
```

Comment:
** Not free **, Normally, docplayer platform is not allowed, we prefer the author, editor or publisher site.

Proof: https://machinelearningmastery.com/machine-learning-with-python/

**Please remove it.**